### ISSUE TITLE
UE4.22.1 - Won't launch [modules missing / try rebuild manually]

### ISSUE THREAD

**Issue by Kuniyo7** [UE4-OceanProject/OceanProject#98]
Hello,

I'm trying to open your project but I keep getting errors about the plugin not being able to compile (and to try building it manually, but I don't know how).
I have also tried an older engine version but I keep getting the same error.

I'm currently using 4.22.1. I only have 4.21.2 as another version installed but I could find the project for that version, so I was unable to test that. The one for 4.22 doesn't load either in 4.21.2 but that isn't that strange.

Can anyone help me with this issue? I'm I doing something wrong?

Kinda new to this... sorry.

Kind regards,

Kuniyo7

**Comment by MarioTennis** [comment]
Hey, I just downloaded the project for the first time and used 4.22 to open it. I got the warning you are mentioning with a popup asking if I wanted to rebuild it and just clicked yes. It did the thing and opened the project fine. It wont work on an older engine from what I can tell since it didn't work when I tried opening it on 4.21 and 4.20. Not sure if this helps but for me it was just a simple pop up asking if I wanted to rebuild.

**Comment by Kuniyo7** [comment]
It turns out there was something wrong with my Visual Studio. I tried building it like you said but it just wouldn't finish building and not even give an error or anything, it would just dissapear.

I re-installed it comepletely and now when I start the project it builds and loads just fine.
I was having issues with another module as well, and I'm guessing that was the same problem for me.

Thanks for reading and hopefully if someone else has this problem they will try the same.

Regards,

Kuniyo7


TL;DR I reinstalled Visual Studio 2017 and right clicked the uproject and let it rebuild, after that the project loads just fine.

**Comment by Wisdemic43** [comment]
Im having the same problem and i was wondering what i have to do

**Comment by NightMare489** [comment]
@Wisdemic43 I guess you are using 4.26
join our discord server and try the fix in #Project-setup. and if have any more question you can ask in #questions
Discord's server invite link : https://discord.gg/Xzrbpup
